dev-scripts/mtlscerts.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+# very much just copied from https://lemariva.com/blog/2019/12/portainer-managing-docker-engine-remotely
+# production use should involve a real external certificate management system
+
+function mtlscerts_command() {
+  export CURRENT=$(pwd)
+  export HOST=${1:-"portainer.p1.alho.st"}
+  export CERTDIR=~/.config/portainer/certs/
```

Comment:
these should be `local`, as you don't need them outside of this script
```suggestion
  local CURRENT=$(pwd)
  local HOST=${1:-"portainer.p1.alho.st"}
  local CERTDIR=~/.config/portainer/certs/
```

and also convention says to name local variables with lower caps, but it's not as important